package.json

```
@@ -147,8 +148,8 @@
     "@types/enzyme": "^3.10.5",
     "@types/jest": "^24.0.6",
     "@types/node": "^10.17.5",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
+    "@types/react": "^18.2.6",
```

It looks like this package has a few new patches we may want to grab
```suggestion
    "@types/react": "^18.2.14",
```

---

Bumped versions of all 4 of these!